core/controllers/admin.py

```diff
@@ -893,21 +1118,37 @@ class ContributorUsersListHandler(base.BaseHandler):
     """Handler to show users with contribution rights."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'category': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': [
+                        constants.CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION, # pylint: disable=line-too-long
```

Comment:
I see this list 3 time till now so I think defining this to a constant is a good idea*